.github/workflows/_run_super_scaffolding_tests.yml

```diff
@@ -0,0 +1,106 @@
+# This workflow will run a few super scaffolding commands and then run tests against the generated code.
+#
+# This workflow is pimarily meant to be called by other workflows, but it can be run manually.
+name: 🏗️ ~ Run super scaffolding tests
+on:
+  workflow_call:
+  workflow_dispatch:
+
+jobs:
+  test:
+    name: "🏗️"
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        # For super scaffolding tests we need to have exactly 7 runners.
+        ci_runners: [TestSite, Project, 'Project::Step', Insight, 'Personality::Disposition', 'Personality::Observation', TestFile]
```

Comment:
What are we basing this off of again? (I forgot)

I know we define a list of constants in the system test file itself... Should we have an explicit explanation in the system test that points to this so we know to change it if necessary?

Reply:
We need exactly 7 runners because we look for node indexes 0-6 in this file:

https://github.com/bullet-train-co/bullet_train/blob/main/test/bin/setup-super-scaffolding-system-test

I'd like to figure out a better way of separating those tests, but probably not as part of this PR.